src/app/myworkflows/my-workflow/my-workflow.component.ts

```diff
@@ -127,9 +127,8 @@ export class MyWorkflowComponent extends MyEntry implements OnInit {
   ngOnInit() {
     this.myWorkflowsService.clearPartialState();
     this.gitHubAppInstallationLink$ = this.sessionQuery.gitHubAppInstallationLink$;
-    this.tokenQuery.gitHubToken$
-      .pipe(takeUntil(this.ngUnsubscribe))
-      .subscribe((token: string) => this.tokenService.getGitHubOrganizations(token));
+    this.tokenQuery.gitHubToken$.pipe(takeUntil(this.ngUnsubscribe)).subscribe();
```

Comment:
I don't think you need this line at all, do you? You're subscribing to it, but not observing anything. Unless there is some side effect I'm missing.

Reply:
You're right, it's not needed. I was planning on removing token-related code in my next PR, but I'll remove this line now since we're here